### PR TITLE
WIP 20 bcarter97 add json return

### PR DESCRIPTION
**Pull request by bcarter97**

### CODE REVIEW

File: test/test_app.py [hunk 1]
```diff
@@ -2,13 +2,19 @@
...
 
-class BasicTestCase(unittest.TestCase):
+class BasicTests(unittest.TestCase):
     def test_index(self):
         tester = app.test_client(self)
         response = tester.get('/', content_type='html/text')
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data, b'Hello World!')
 
+    def test_json(self):
```
Comment (georgeracu) [inline test/test_app.py]: Maybe is better to split these into two test classes, such that we can put together all tests that belong to a certain endpoint (eg `/EUS` or `/`).

File: scripts/win-dev.bat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+@ECHO Install the prereqs
+pip install virtualenv
+virtualenv venv
+
+@ECHO Start the virtual environment
+call venv/Scripts/activate
+
+@ECHO Install dependencies in the venv
+pip install -r ./requirements/common-requirements.txt
+pip install -r ./requirements/dev-requirements.txt
```
Comment (georgeracu) [inline scripts/win-dev.bat]: `dev-requirements.txt` contains all the others, so there is no need to install `common` and `test` by hand. Only dev should be enough.

File: test/test_app.py (hunk 1 above)
Comment (bcarter97) [reply test/test_app.py]: done

File: scripts/win-dev.bat (hunk 2 above)
Comment (bcarter97) [reply scripts/win-dev.bat]: I didn't realise they had a built in hierarchy, thanks

**Comment by bcarter97** [comment]
- Changed tests into individual files
- Made JSON test easier to understand

**Pull request merged**